engines/godot/assets/run-godot.sh

```diff
@@ -1,4 +1,12 @@
 #!/bin/bash
 
-echo "launching game in Godot"
-./godot.x11.tools.64 --fullscreen --main-pack *.pck
+# Find pck file
+pckfile=$(find . -name \*.pck)
+
+# Check if pck file exists and launch game if it does.
+if [ -n "$pckfile" ]; then
+    echo "Launching .pck file from $pckfile"
+    ./godot.x11.tools.64 --fullscreen --main-pack "$pckfile"
+else
+    echo "Couldn't find .pck file"
+fi
```

Comment:
Are you wanting this to show to the user? If so, look at how some of the other packages show errors, such as the source sdk. Let me know if you have any questions

Reply:
Like this?

Reply:
@ToughGuyKunio That isn't quite right, as it only uses zenity, there's a way to use egui as a start, with zenity as a fallback, that should be used:

For example, look at this: https://github.com/luxtorpeda-dev/packages/blob/master/engines/source-sdk-2013/assets/downfall/run-downfall.sh

At the start of the script, you will need this:

if [ -f "last_error.txt" ]; then
    rm last_error.txt
fi

Then in the area that you want to do the error, do this:

 error_message="Path to Source SDK 2013 incorrect."
            if [[ -z "${LUX_ERRORS_SUPPORTED}" ]]; then
                "$STEAM_ZENITY" --error --title="Error" --text="$error_message"
            else
                echo "$error_message" > last_error.txt
            fi
            exit 10

Reply:
I see, updated now

Reply:
@ToughGuyKunio Looks good thanks